bin/.eslintrc.js

```diff
@@ -6,15 +6,6 @@
 'use strict';
 
 module.exports = {
```

Comment:
Question: Why do we need an extra `eslintrc.js` in bin, misc folders, which does not inherit from the root file?

Reply:
They **all** inherit from the root eslint file (except for the test eslint configs). At some point, eslint changed the logic around inheritance a bit, now it always implicitly inherits from all eslint configs it finds when going _upwards_ in the directory hierarchy:

https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy

The only exception is when you explicitly tell it to `extend` from a specific path, then it will extend from that path and from everything in the directory hierarchy. 

Last but not least, with `extends` and `root: true`, it will only extend from the explicity provided `extends` path.

The reason for the config files in bin and misc are the few rules I explicitly add there:
* `'no-console': 'off'` for scripts in bin and
* `'import/no-unresolved': 'off'` for misc, because on CI we do not run `npm install` in those locations. (no-unresolved compares what is installed in node_modules with your require statements, and there is no node_modules folder if we don't run `npm install`)

Reply:
Ah ok 👌🏻